.pre-commit-config.yaml

```
@@ -10,7 +10,7 @@ repos:
     -   id: trailing-whitespace
     -   id: check-ast
     -   id: check-merge-conflict
-    #-   id: flake8
+    -   id: flake8
```

Can you force python version (as it doesn't pass on python3.6)?
```suggestion
    -   id: flake8
        language_version: python2.7
```